storm-core/test/clj/backtype/storm/supervisor_test.clj

```diff
@@ -261,8 +260,8 @@
                                 (str "-Dstorm.id=" mock-storm-id)
                                 (str "-Dworker.id=" mock-worker-id)
                                 (str "-Dworker.port=" mock-port)
-                               "-Dstorm.log.dir=/logs"
-                               "-Dlog4j.configurationFile=/log4j2/worker.xml"
+                                (str "-Dstorm.log.dir=" file-path-separator "logs")
+                                (str "-Dlog4j.configurationFile=" file-path-separator "log4j2" file-path-separator "worker.xml")
```

Comment:
Seems like there're many pull requests regarding log4j configuration applied without backporting to 0.10.x. STORM-1463 is just one of these.

I'm wondering we would want to find whole of things and backport to 0.10.x. If we don't want, this PR is just good to go for 0.10.x, and I'll create pull request for 1.x and master.

What do you think?